Modules/_testcapi/parts.h

```diff
@@ -1,7 +1,9 @@
-#include "Python.h"
-
 /* Always enable assertions */
-#undef NDEBUG
+#ifdef NDEBUG
+#  undef NDEBUG
```

Comment:
Thank you for catching this.
I'm moving the `#undef` in #95992. Since this change can't be backported to 3.11 (the file doesn't exist there), could you remove it from this PR?

Reply:
Done